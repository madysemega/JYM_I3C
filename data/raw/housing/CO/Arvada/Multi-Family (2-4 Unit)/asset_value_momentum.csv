predictions_asset_value_momentum
-0.13647654930336997
0.15390994305590902
-0.02704366099012518
0.175106677013146
-0.25851583948035495
-0.03225163526940798
-0.2436400795558712
-0.10502280474955492
-0.14299239811747874
-0.06995652010916084
-0.2180145449301055
-0.05593300098795055
-0.2124941416722953
0.3199837941975297
-0.3031086472504451
0.12602375246152936
0.07320591600907264
1.6523684550872548
-0.12655108624683092
-0.6179115717014811
-0.1760614766607128
0.07907377362370735
0.18783386651976208
-0.05392392445589061
-0.26821201000462197
-0.20722705392926943
-0.6683297574220074
