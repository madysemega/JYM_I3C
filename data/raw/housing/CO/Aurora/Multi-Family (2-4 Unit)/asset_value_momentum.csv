predictions_asset_value_momentum
-0.0992370366047184
0.018487979388632024
0.295983265022324
0.3334414918082464
0.5469469828816818
0.5156564830420265
0.925877376276386
1.1947394335496324
0.08579739313071084
-0.15398900971779303
0.27061850037270946
-0.04473370519236858
0.0827768834856612
-0.17598512219233417
1.6736605998720626
1.577111794067676
1.0521147351245916
0.4101593905108779
0.4205217148134188
1.3379622406756952
0.01023592253565637
-0.14912152627630393
0.17644369716110497
1.5494643600878915
1.5306429752658743
0.41713349910868003
0.06138586077420901
