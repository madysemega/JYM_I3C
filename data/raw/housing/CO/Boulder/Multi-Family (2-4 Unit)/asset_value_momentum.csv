predictions_asset_value_momentum
0.12001189782350065
0.31436105848592444
0.20517714852146252
-0.36725911608376616
0.5069978820172912
-0.14442267642073783
0.026984298730185548
0.32953015054507884
0.596892478507792
0.14800825381280053
0.4546415905118408
-0.3979952822869584
0.5913133573525952
0.28774034482634225
0.45322617288087635
0.328497158899505
0.192610402402101
-0.3247804479619182
-0.08420578580019844
0.7798807718958789
-0.23186009640861407
0.33117355770381807
-0.20101283008387644
0.1978006835442717
0.4328430594473935
-0.1684517480697632
0.03503948444990805
