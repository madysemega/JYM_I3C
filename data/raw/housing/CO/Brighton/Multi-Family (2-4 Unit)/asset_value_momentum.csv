predictions_asset_value_momentum
-0.1595069544801351
3.1354853045109228
-0.7000848356767685
0.7393120017789621
2.1197097098865814
-0.24081424225667936
0.33186126811771366
0.039241891219506886
0.051713125665668706
0.07218285400486994
0.3143844829706463
0.43249295562725587
0.311802108648803
0.3129136514278638
-0.07750979072833171
0.3598760604950689
1.319466261938558
-0.4932020247577782
1.561172693457318
0.8319355735417803
-0.7444639410385357
0.49959510718797995
-0.29056321668840884
1.0571006171665582
-0.1514083605070022
2.3484010535475086
1.5153256540427984
