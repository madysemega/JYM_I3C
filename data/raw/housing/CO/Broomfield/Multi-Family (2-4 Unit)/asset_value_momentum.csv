predictions_asset_value_momentum
0.013533932858965113
-0.9657425361578412
-0.02633544752032077
-0.010324504169324455
-0.8490489011218967
0.04594509646559179
0.02011550433702922
-0.8127847179597157
-0.08778740258740361
-0.00015597827564528617
0.060342921551944234
0.3669564471016746
-0.8366756190149376
-0.2724744941172994
0.029753881596998065
-0.34115132712073326
-0.33660636335020433
-0.07302029190923184
0.011360416714690104
-0.5245549166271208
-0.48925606471102673
-0.935251296269668
0.4087654510936744
0.3346853975358834
-0.32712765431605445
-0.8427219566055215
0.0015314517203069296
