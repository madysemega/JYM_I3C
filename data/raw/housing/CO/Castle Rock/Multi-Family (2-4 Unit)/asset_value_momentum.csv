predictions_asset_value_momentum
0.03499126503708279
0.2327239470382521
0.0018819016854999468
-0.03453727722592148
-0.00604872523173919
0.03928092879771137
0.01038194473043816
-0.008957033410847378
0.4115495644981876
0.033573588033164965
0.05442260336675035
0.1911751455293885
0.06296176977238523
-0.3926533744647332
0.8645238818882818
-0.030252387137950847
-0.035326178520393575
0.24508464595755902
0.014389754253262717
0.03135766994044428
-0.5157484132091106
0.06920023350787456
0.03473909059015209
0.7792335027126756
0.014895971105990273
0.19211078537034618
0.773826176897132
