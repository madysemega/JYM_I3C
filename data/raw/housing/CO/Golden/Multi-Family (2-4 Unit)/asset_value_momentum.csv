predictions_asset_value_momentum
-0.5014106129818489
-0.007787246116576639
-0.06064473702400967
0.005816608866176623
0.0030840960034490333
0.035858652793757426
-0.1266763422639494
-0.24607205465361123
-0.24650184826671834
-0.01764374038477833
0.06131138541312511
0.050509662973250914
-0.5131462295412447
0.05939900244552547
-0.08724112317198217
-0.2313586277429968
-0.22533177199316204
-0.030171231153215447
-0.1433026023264779
0.09511514299045798
0.022620029697139287
-0.03367506863510326
-0.24158252537461486
-0.29634533532812585
-0.38684619258975134
-0.24261773744630816
-0.09487646099576176
